trio/_dtls.py

```diff
@@ -754,6 +798,19 @@ async def dtls_receive_loop(endpoint_ref, sock):
 
 @attr.frozen
 class DTLSChannelStatistics:
+    """An object with statistics about this connection.
+
+    Currently this has only one attribute:
+
+    - ``incoming_packets_dropped_in_trio`` (``int``): Gives a count of the number of
```

Comment:
I _think_ that Sphinx will do the right thing if we move this description to an attribute-docstring below, and set `:members:` in the .rst file.

Reply:
It seems that *should* work, either with `:members:` or with `:autoattribute:` but I seem to fail to find the right magical invocation to get it to work and print the attribute docstring. I've tried setting the attribute-docstring both with `#:` on the line before, or with a `"""` docstring afterwards. Anybody else willing to play around with it is very welcome to.


https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute
https://stackoverflow.com/questions/22717179/sphinx-class-attribute-documentation